code/modules/meteors/meteor_types.dm

```diff
@@ -42,6 +42,7 @@
 	SSaugury.register_doom(src, threat)
 	SpinAnimation()
 	chase_target(target)
+	QDEL_IN(src, lifetime)
```

Comment:
A: this breaks my haha funny admin event (pause the movement ss spam meteors then turn it on)
B: timer spam man bad
would be best to hook into the qdeletion of its loop

Reply:
That was my initial idea but I thought it was too niche. Sure.